package.json

```diff
@@ -32,7 +32,8 @@
         "*.js": [
             "npm run lint:fix",
             "npm run prettier",
-            "git add ."
+            "git add .",
+            "npm test"
```

Comment:
Good job. Implementing a precommit hook for running the test files before every commit.